content/_code-samples/send-a-memo/js/send-a-memo.js

```diff
@@ -0,0 +1,58 @@
+if (typeof module !== "undefined") {
+    // Use var here because const/let are block-scoped to the if statement.
+    var xrpl = require('xrpl')
+  }
```

Comment:
nit: Spacing to line up with the if-statement.
```suggestion
}
```